ckanext/switzerland/commands.py

```diff
@@ -15,6 +17,10 @@ class OgdchCommand(ckan.lib.cli.CkanCommand):
         paster ogdch help
         # Cleanup datastore
         paster ogdch cleanup_datastore
+        # Cleanup harvester jobs and objects:
+        # - deletes the all harvest jobs and objects except the latest
+        # - how many jobs to keep can be set in the ckan config
```

Comment:
Now that we are taking the paster-command approach, I think I'd prefer to define a default for the number of jobs to keep here and instead of configuring it in the ini-file, to pass it as a variable with the command.
What do you think about it?

Reply:
Okay, a default value is fine for me as well. 
- Should I then remove the configuration? Or do you want both default value and configuration?
- I could also move getting the configuration into the logic function instead of getting it in the command?

Reply:
Yeah, I'd suggest to remove the configuration-option and using a default value that you can define here. (Maybe a little bigger than 2 but I am assuming that is for testing?) But passing it with the command would also make it easier to debug.

Reply:
Yes, I totally agree and implemented according to your suggestion.